cloud/cluster/node.go

```diff
@@ -3,7 +3,28 @@ package cluster
 type NodeId string
 
 type Node interface {
+	// A unique node identifier, like 'host:port'
 	Id() NodeId
+
+	// Comparator for sorting.
+	Less(Node) bool
```

Comment:
I think just because that's how it was stored in that iteration. Is there an example where sorting by addr then port is different than sorting by ID?